.github/scripts/group_bash_logs.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+# Create an expandable group in the logs for the bash command output using the syntax defined for GiHub Action runners:
+# https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines.
+# With such expandable logs groupings we can avoid noisy outputs and add the same time have pointers for
+# debugging/analysis.
+function run_and_group_bash_logs {
```

Comment:
I'd just make "print_df.sh" script :). No need to generalize it. It would be quite a bit simpler.

Reply:
addressed, please check if that's what you meant to suggest